outofcore/include/pcl/outofcore/cJSON.h

```diff
@@ -44,23 +44,37 @@ extern "C"
 #define cJSON_IsReference 256
 
 /* The cJSON structure: */
-typedef struct cJSON {
+using cJSON = struct cJSON {
```

Comment:
Please undo the changes in this file. Simply using `struct cJSON` (by removing the typedef and cJSON at the end) will suffice for a C++ header. `using` is not valid C (which I think this file was originally in.